src/hooks/settings.ts

```diff
@@ -5,7 +5,7 @@ import db from '../dal/storage';
 export const defaultSettings: Settings = {
     themeForegroundColor: 'white',
     themeBackgroundColor: 'black',
-    themeBackgroundImagePath: undefined,
+    themeBackgroundImageBase64: undefined,
```

Comment:
For discussion: do you think better to store the base64 somewhere else?
For example, use db.load('backgroundImage').
Then, when we open the devtools, the settings will still be easy to read